src/cortxfscli/cortxfscli.py

```diff
@@ -30,6 +30,7 @@
 import textwrap
 import traceback
 import re
+import hashlib
```

Comment:
Is the library available with basic python package/rpm ? or it needs any extra rpm to be installed ?

Reply:
It's available with the basic python package. No need to install any extra rpm.